### PR TITLE
feat: pipelines config doc page

### PR DESCRIPTION
**Pull request by ZachGoldberg**
image
image

### CODE REVIEW

File: _docs-sources/pipelines/overview/configuration.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  logs-account-name: logs
+  # (Optional) override the folder for the management account, default is 'management'
+  management-account-name: management
+
+
+  ###########################################################################
+  # Configuration options below are ONLY for Pipelines Enterprise customers
+  ###########################################################################
+
+  # Only set to true for delegated (vended) accounts from account factory
```
Comment (yhakbar) [inline _docs-sources/pipelines/overview/configuration.md:53]:
```suggestion
  # Only set to true for delegated (vended) repositories from account factory
```

File: _docs-sources/pipelines/overview/configuration.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  is-delegated-repo: false
+  # Only set to true for infra-live-access-control
```
Comment (yhakbar) [inline _docs-sources/pipelines/overview/configuration.md:55]:
```suggestion
  # Only set to true for infrastructure-live-access-control
```

File: _docs-sources/pipelines/overview/configuration.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  is-access-control-repo: false
+
+  account-vending:
+    sdlc:
+    # Map of account identifiers to account vending configurations.
+    # A valid account vending configuration includes:
+    # - account-identifiers: List of account identifiers that will be created (required).
```
Comment (yhakbar) [inline _docs-sources/pipelines/overview/configuration.md:62]: I think it's confusing to have what looks like valid YAML list elements under `sdlc` in this comment. Consider placing each comment right above the relevant configuration.

**Review by yhakbar — CHANGES_REQUESTED** [review]

**Comment by ZachGoldberg** [comment]
@yhakbar mind re-reviewing?

**Pull request merged**